overlord/snapshotstate/backend/backend.go

```diff
@@ -336,7 +336,8 @@ func Save(ctx context.Context, id uint64, si *snap.Info, cfg map[string]interfac
 	w := zip.NewWriter(aw)
 	defer w.Close() // note this does not close the file descriptor (that's done by hand on the atomic writer, above)
 	savingUserData := false
-	if err := addDirToZip(ctx, snapshot, w, "root", archiveName, si.DataDir(), savingUserData, snapshotOptions.ExcludePaths); err != nil {
+	baseDir := snap.BaseDataDir(si.InstanceName())
```

Comment:
Nitpick^2, maybe `baseDataDir`? baseDir sounds like dir of a base snap.

Reply:
Done. Thanks